catalog.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>NЁRDS</title>
+</head>
+<body>
+	<header>
+		<nav>
+			<a href="index.html">
+				<img src="https://via.placeholder.com/150x100" alt="">
+			</a>
+			<ul>
+				<li><a href="studio.html">Студия</a></li>
+				<li><a href="clients.html">Клиенты</a></li>
+				<li><a href="catalog.html">Магазин</a></li>
```

Comment:
```suggestion
				<li><a>Магазин</a></li>
```
Это текущая страница, а значит `href` ссылке не нужен.